kratos/sources/kratos_application.cpp

```diff
@@ -160,26 +174,28 @@ void KratosApplication::RegisterKratosCore() {
     //Register specific conditions ( must be completed : conditions defined in kratos_application.h)
     //generic condition
     KRATOS_REGISTER_CONDITION("GenericCondition", mGenericCondition);
-    //point conditions
+    // Point conditions
```

Comment:
```suggestion
    
    // Point conditions
```
If we are to change it lets make it consistent with the ones below...